include/bluetooth/services/hids_c.h

```diff
@@ -220,340 +195,364 @@ struct bt_gatt_hids_c_info_val {
 };
 
 /**
- * @brief HIDS client initialization parameters
- *
- * The parameters for initialization function.
+ * @brief HIDS client parameters for the initialization function.
  */
 struct bt_gatt_hids_c_init_params {
 	/**
-	 * @brief Function that would be called when HID client is ready
+	 * @brief Function to call when the HID client is ready.
 	 */
 	bt_gatt_hids_c_ready_cb ready_cb;
 	/**
-	 * @brief Function that would be called if HID preparation fails
+	 * @brief Function to call if HID preparation fails.
 	 */
 	bt_gatt_hids_c_prep_fail_cb prep_error_cb;
 	/**
-	 * @brief Function that would be called when protocol mode is updated
+	 * @brief Function to call when the protocol mode is updated.
 	 *
-	 * This function would be called every time protocol mode is read or
-	 * written. Old value is not checked.
+	 * This function is called every time the protocol mode is read or
+	 * written. The old value is not checked.
 	 */
 	bt_gatt_hids_c_pm_update_cb pm_update_cb;
 };
 
 /**
- * HIDS client structure
+ * @brief Report base information.
+ *
+ * This structure is common for Input and Output/Feature reports.
+ * Some of the fields are used only for the Input report type.
+ *
+ */
+struct bt_gatt_hids_c_rep_info;
+
+/**
+ * @brief HIDS client object.
+ *
+ * @note
+ * This structure is defined here to allow the user to allocate the memory
+ * for it in an application-dependent manner.
+ * Do not use any of the fields here directly, but use the accessor functions.
+ * There are accessors for every field you might need.
  */
 struct bt_gatt_hids_c {
-	/** Connection object */
+	/** Connection object. */
 	struct bt_conn *conn;
-	/** HIDS client information structure */
+	/** HIDS client information. */
 	struct bt_gatt_hids_c_info_val info_val;
-	/** Handlers for descriptors */
 	struct {
-		u16_t pm;      /**< Protocol Mode characteristic value handle */
-		u16_t rep_map; /**< Report Map descriptor handle              */
-		u16_t info;    /**< HID information characteristic handle     */
-		u16_t cp;      /**< HID control point characteristic handle   */
+
+                /** Protocol Mode Characteristic value handle. */
+		u16_t pm;
```

Comment:
The doc build fails because Sphinx cannot distinguish between handlers::pm and pm (line 324/317). One of them must be renamed.